Backend/WebAPI/Controllers/UsersController.cs

```diff
@@ -65,6 +71,45 @@ public string Register([FromBody] RegisterRequest request)
             return "got to end without stuff";
         }
 
+        /// <summary>
+        /// NOT DONE. STILL IN PROCESS
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("api/users/login")]
+        public IHttpActionResult Login([FromBody] LoginRequest request)
+        {
+            IUserService _userService = new UserService();
+            IPasswordService _passwordService = new PasswordService();
+
+            using (var _db = new DatabaseContext())
+            {
+                User user = _userService.GetUser(_db, request.email);
+                //PasswordReset ps = ps.
+                bool isPasswordMatched = _passwordService.VerifyPassword(request.password, user.PasswordHash, user.PasswordSalt);
+
+                //succesful login
+                if (isPasswordMatched == true)
```

Comment:
This needs to return a token via a created session

Reply:
if statement now returns token through a created session, I believe.